pacai/bin/arguments.py

```diff
@@ -25,6 +21,10 @@ def getParser(description, name):
             action = 'store_true', default = False,
             help = 'set logging level to warning (default: %(default)s)')
 
+    parser.add_argument('-s', '--seed', dest = 'seed',
+            action = 'store', type = int, default = 0,
```

Comment:
This default actually ensures that there will be a fixed seed by default.
(So you accidentally fixed the seed.)

Instead, lets create a random seed (you can just use an unseeded `random.randint(0, 2 ** 32)`).
So, set the default to None (-1 if you can't use None).
Then before seeding check to see if the seed is None, gen the seed, and seed with it.

Also regardless of what seed it used, do a debug log telling people what the seed it.
This makes it much easier to debug things when they go wrong (since we can just set the seed to whatever it was when there was the issue).